src/lib/Config.js

```diff
@@ -58,6 +58,20 @@ class Config {
             }
         }
 
+        this.healthChecks = { allowFrom: ['127.0.0.1/8', '::1'] };
+        if (config.healthChecks && config.healthChecks.allowFrom) {
+            assert(config.healthChecks.allowFrom instanceof Array,
```

Comment:
I don't know if using Array.isArray rather than instanceof matters in node but it is the preferred method at least for web.